rpp-charter.md

```diff
@@ -1,50 +1,34 @@
-New functionalities, not having any equivalents in EPP, may be defined for RPP.
 
-The RPP WG considers RPP extensions that are functional equivalents of registered [EPP extensions](https://www.iana.org/assignments/epp-extensions/epp-extensions.xhtml) in the construction of requirements.
+New functionalities, not having any equivalents in EPP or [EPP extensions](https://www.iana.org/assignments/epp-extensions/epp-extensions.xhtml), may be defined for RPP.
 
 The RPP working group is focused on designing a new protocol intended to co-exist alongside EPP, supporting diverse needs in the ecosystem. 
 As such, considerations for replacing EPP or migration scenarios away from EPP are outside the scope of the RPP WG.
 
 The REGEXT working group is chartered to maintain and standardize extensions to EPP. 
 Consequently, any extensions or changes to EPP, including those related to RPP functionality that do not exist in EPP, are explicitly out of scope for the RPP working group. 
 
-# Working Group Process
-
-The charter in datatracker is the source of truth for all official working group process requirements.
-
-However, the RPP WG publishes its usage of tools beyond the datatracker and mailing lists on its wiki.
-
-[BCP 56](https://datatracker.ietf.org/doc/html/rfc9205) is used to guide the specification of RPP, along with [RFC 6570](https://datatracker.ietf.org/doc/html/rfc6570), [RFC 8288](https://datatracker.ietf.org/doc/html/rfc8288), [RFC 9421](https://datatracker.ietf.org/doc/html/rfc9421), [RFC 9652](https://www.rfc-editor.org/rfc/rfc9652.html), and the outputs of the [httpapi working group](https://datatracker.ietf.org/wg/httpapi/about/).
-
-Analysis of the functionality included and commonly used in core EPP for domain names, hosts and contacts ([RFC5730](https://datatracker.ietf.org/doc/html/rfc5730), [5731](https://datatracker.ietf.org/doc/html/rfc5731), [5732](https://datatracker.ietf.org/doc/html/rfc5732), and [5733](https://datatracker.ietf.org/doc/html/rfc5733)) is developed to support the deliverables. 
-
-In alignment with the [IESG's Statement on Support Documents dated 2023-08-24](https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/), these supporting documents are publicly available on the working group's wiki.
-
-Chairs record the justification for a document being in scope for the charter at the time of working group adoption on the mailing list.
+RPP resources are secured with appropriatly strong authentication, only encrypted transports are used to properly safeguard authentication material.
 
-All consensus decisions are conducted on the mailing list.
+Security requirements for RPP are intentionally generic, enabling operators to choose the specific authentication mechanisms and degree of automation that is supported.
```

Comment:
This is putting a requirement "enabling operators to choose the specific authentication mechanisms" into a charter, which we may not want to have. It's always a choice between flexibility/freedom of choice and potential risk of fragmentation.
Would the sentence above be not enough?

Reply:
Im not sure what you mean, lets wait to hear back from the folks who made comments before proposing alternatives.